### ISSUE TITLE
Incompatible with embedded presentations

### ISSUE THREAD

**Issue by Gkleinereva** [satoshifuku/handwriting#9]
Reveal.js supports embedded presentations (https://revealjs.com/initialization/#multiple-presentations) in order to allow for multiple presentations to be displayed on the same page.  In order to accomplish this, Reveal can be used as a constructor to create multiple Reveal objects.  This plugin doesn't take this use case into account (the code references the global Reveal object to pull configuration, among other things).  Thus, the plugin doesn't work if reveal has been initialized with the following pattern:

`
let presentation1 = new Reveal(document.querySelector('#presentation1', {...});
`

My partner and I are interested in using the plugin with the above use case - would you mind a pull request fixing the issue?  Thanks!

**Comment by Gkleinereva** [comment]
Hey there @satoshifuku any thoughts on this?  Thanks!

**Comment by satoshifuku** [comment]
Sorry for the latency here.
I've had some personal matters that have been keeping me busy.

I'm glad you are interested in my work.
I never mind. i wait your pull request!

**Comment by Gkleinereva** [comment]
Thank you!  It may be some weeks or months before I am able to actually make the change, but I will get to it.  I will keep you posted if I have any questions of interest to you.

**Comment by Gkleinereva** [comment]
Hello!  I've spent quite a bit of time reviewing your codebase, and I'm not sure making changes that I want to this repository will work well for everyone currently using it.  I'm really looking to do 3 things:

1) Remove FontAwesome dependency
2) Remove icon dependencies
3) Update the presentation to be an ES6 module (thereby allowing it to be embedded)

I'm worried that this would break backwards compatibility for folks currently using it, and that my changes are too custom to make in such a way that they would be dynamic for everyone.  So I think I'll just modify a local version of handwriting.js for our use if that's ok with you.  Thank you so much for your time!  I'll close this issue.

**Comment by satoshifuku** [comment]
Thank you for spending your time!
I will consider modifying 3 things.

**Comment by Gkleinereva** [comment]
If you want a hand making the changes, I would be happy to share what I'm doing with you when it's done!

**Comment by satoshifuku** [comment]
Of course, I am happy if you share it.
And if you do not mind, creating the pull request.

**Comment by Gkleinereva** [comment]
I'm not quite done tweaking things, but I don't think you will want to incorporate my updated version directly, unfortunately.  I made quite a few changes that wouldn't apply to everyone's use case.  For example

1. I replaced the icons with icons from a library we're using
2. I changed how the buttons appear and function

So I'm not sure if a Pull Request is the right method.  I could potentially replicate some of my changes in your repository if you want (related to embedded presentations).  Give me a few weeks to polish things and then we can revisit this?
